src/components/Toolbar/styles.ts

```diff
@@ -26,11 +27,39 @@ export const RunButton = styled(Button)`
     letter-spacing: 0.3px;
     justify-content: flex-start;
     border-radius: 0px;
+    margin: 2px;
   }
 `;
 
 export const RunCodeIcon = styled(PlayArrowIcon)`
-  color: ${btnColor};
+  color: ${runBtnColor};
+  &&& {
+    font-size: 26px;
+  }
+`;
+
+const nextBtnColor = "#36454C";
+export const NextButton = styled(Button)`
+  width: 180px;
+  color: ${nextBtnColor};
+  &&& {
+    border: 4px solid ${nextBtnColor};
+    color: ${nextBtnColor};
+    background-color: white;
+    text-align: center;
+    display: flex;
+    vertical-align: center;
+    padding: 6px;
+    outline: none;
+    font-size: 14px;
+    letter-spacing: 0.3px;
+    border-radius: 0px;
+    margin: 2px;
+  }
+`;
+
+export const NextStepIcon = styled(SkipNextIcon)`
+  color: ${nextBtnColor};
```

Comment:
Przyznaję się bez bicia - skopiowałam. Powinnam może zrobić jakiś template dla buttona, żeby zmieniać dla nich jedynie kolory a pozostałe właściwości i tak są takie same. Zrobić to? :D

Reply:
![](https://media.giphy.com/media/lp84cYEUoOyh0LBEbt/giphy.gif)

To mogło by wyglądać jakoś tak 
```ts

const BaseButton = (color: string) => styled(Button)`
  width: 180px;
  color: ${color};
  &&& {
    border: 4px solid ${color};
    color: ${color};
    background-color: white;
    text-align: center;
    display: flex;
    vertical-align: center;
    padding: 6px;
    outline: none;
    font-size: 14px;
    letter-spacing: 0.3px;
    border-radius: 0px;
    margin: 2px;
  }
`;

export const RunButton = BaseButton(runButtonColor);
export const NextStepIcon = BaseButton(runButtonColor);

```